x/incentives/keeper/distribute_test.go

```diff
@@ -92,6 +93,97 @@ func (suite *KeeperTestSuite) TestDistribute() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestDistributeToCLPools() {
+	fiveKRewardCoins := sdk.NewInt64Coin(defaultRewardDenom, 5000)
+	fifteenKRewardCoins := sdk.NewInt64Coin(defaultRewardDenom, 15000)
+
+	coinsToMint := sdk.Coins{sdk.NewInt64Coin(defaultRewardDenom, 1000000)}
+
+	tests := map[string]struct {
+		// setup
+		numPools int
+
+		// expected
+		expectErr             bool
```

Comment:
Are there any error case we can test here?

Reply:
there's really one thing i can think of that we can check for invalid case; 
1. Amount cannot be <=0 here because `for _, coins := gauge.Coins` will only run for position coins 
2. Same thing ^ goes for emissionRate because we use coinAmt to calculate it 
3. I can check minUptime but it gets hardcoded as epoch value when pool gets created aka gauge creation 

the only thing to check would be if  starttime < currentBlockTime